### PR TITLE
Update content-security-policy for new UI

### PR DESCRIPTION
**Pull request by C-gyorfi**
Currently, the new UI crashes when running behind the Nginx-auth-proxy because some of the client site JS violates the content security policy
image

### CODE REVIEW

File: Nginx_Auth_Proxy/confd/templates/bichard.https.conf.tpl [hunk 1]
```diff
@@ -88,12 +88,13 @@ location /bichard {
...
     auth_request_set $auth_cookie $upstream_http_set_cookie;
     add_header  Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
-    include /etc/includes/headers.conf;
+    include /etc/includes/ui-headers.conf;
     add_header Set-Cookie "$auth_cookie; secure";
 
     proxy_pass        https://$ui;
     proxy_ssl_verify  {{ getv "/cjse/nginx/proxysslverify" "on" }};
     proxy_set_header Host $http_host;
+    proxy_pass_header Content-Security-Policy;
```
Comment (tomvaughan77) [inline Nginx_Auth_Proxy/confd/templates/bichard.https.conf.tpl:97]: Adding the CSP to the response header is now handled by the UI itself

Comment (jaskaransarkaria) [reply Nginx_Auth_Proxy/confd/templates/bichard.https.conf.tpl:97]: I think a small comment explaining this change would be handy

**Comment by jaskaransarkaria** [comment]
I think it's worth adding a test for this change see the `test/` dir

File: Nginx_Auth_Proxy/confd/templates/bichard.https.conf.tpl (hunk 1 above)
Comment (tomvaughan77) [reply Nginx_Auth_Proxy/confd/templates/bichard.https.conf.tpl:99]: Done 💯

**Review by jaskaransarkaria — APPROVED** [review]

File: Nginx_Auth_Proxy/includes/ui-headers.conf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+add_header  Cache-Control "no-store, no-cache, must-revalidate";
```
[the remark this answers is not in the document]
Comment (tomvaughan77) [reply Nginx_Auth_Proxy/includes/ui-headers.conf:1]: Absolutely no idea, but it's included on all the endpoints so I'm assuming there's a good reason?

**Pull request merged**